docs/usage.rst

```diff
@@ -133,6 +133,26 @@ would rather see the equivalent XRootD endpoints for the files:
 
 The data files can be downloaded via XRootD protocol using the **xrdcp** command.
 
+**Verbose**
+
+Note that you can use ``--verbose`` command-line option if you would rather want to see
```

Comment:
Let's revert the order:

If you would like to know in advance the file sizes and checksums, you can use `--verbose` option: